requirements/env_common.yml

```diff
@@ -27,3 +27,4 @@ dependencies:
     - xgboost
     - gensim  # used by mol2vec
     - tensorboard
+    - triton
```

Comment:
Is triton a must requirement?

Reply:
Since the model compilation is done by PyTorch, probably, we can skip it in dependency and test only for one or two modes - thus testing only the API. If the user wants it, they can have it installed.

Reply:
triton is needed for testing the modes `reduce-overhead` and `max-autotune`. We can remove the test for these two modes and only do testing for `default` and `max-autotune-no-cudagraphs` which doesn't require any additional dependency. Shall I remove the tests for `reduce-overhead` and `max-autotune`?

Reply:
yes